pandas/io/excel/_odfreader.py

```diff
@@ -0,0 +1,164 @@
+import pandas as pd
+
+from pandas.io.parsers import TextParser
+
+
+class ODFReader(object):
```

Comment:
Should remove `(object)` here - we recently get rid of the Python2 "new style" classes so keeping this is what is causing your CI failure

Reply:
Thanks for the hint...

It first complained that I had a leading underscore in _ODFReader which i fixed, but I guess there were more lint checks hiding behind that one.